CHANGES.rst

```diff
@@ -25,6 +25,7 @@ Bugs fixed
 
 * #13369: Correctly parse and cross-reference unpacked type annotations.
   Patch by Alicia Garcia-Raboso.
+* #13550: Fix typographical mistake for `linkcheck_allowed_redirects`.
```

Comment:
```suggestion
```

Reply:
LOL, we made the same mistake. It needs rst syntax, not MyST. Sorry about that.
```suggestion
* #13550: Fix typographical mistake for ``linkcheck_allowed_redirects``.
```

Reply:
No, my suggestion was to remove the changes entry, this is a doc-only typo fix rather than a user-observable behavioural change.

Reply:
Oh, I gotcha, I misinterpreted the PR template.

https://github.com/sphinx-doc/sphinx/blob/f928da16337699ece42f5d7fc377870422eee3e3/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L17

Should I update that to be explicit with "user-observable behavioural"?

Reply:
I think user-visible and user-observable are synonyms? The rule of thumb is that if it changes something in `sphinx/` that isn't a refactor, it should have a CHANGES entry. I tend to avoid wanting to add more, though, as CHANGES is also our release notes -- we don't have *What's New* or etc here.

Reply:
Yeah, they're synonymous. Either way, I'm suggesting adding the word "behavioral" to the term in the PR template, which makes the intent clear to me.